lib/paperclip/matchers/validate_attachment_content_type_matcher.rb

```diff
@@ -65,7 +65,7 @@ def accepted_types_and_failures
         end
         def rejected_types_and_failures
           if @rejected_types.present?
-            "Reject content types: #{@rejected_types.join(", ")}\n".tap do |message|
+            "Reject content types: #{@rejected_types.join(", ")}\n".dup.tap do |message|
```

Comment:
Line is too long. [88/80]<br>Prefer single-quoted strings inside interpolations.